# The 'value' field is either 0 (no fault) or 1 (hardware error).

Header header
uint32 value